agendas/2024/sprint-kickoff/2024-03-01-Flow-Sprint-Kickoff.md

```diff
@@ -273,12 +273,21 @@ Cycle Objective(s): Transaction fees on EVM, increasing transaction fees and inf
 
 **Done last sprint**
 
-*
+* EVM to gas computation ratio set at 1000:1 for PreviewNet launch. Evaluating a higher 3000:1 ratio based on Flow's ability to execute higher gas transactions.
+* Built developer docs (and internal doc) on FlowEVM gas fee
```

Comment:
@KshitijChaudhary666 mind adding links to those docs incase someone wants to go over. (Also a good opportunity to share these links)